### DISCUSSION TITLE
Collecting a listing of AI for EO papers

### DISCUSSION THREAD

**Discussion by brunosan** [Clay-foundation/model#179] [Related work]
@calebrob6 has compiled this list of relevant AI for EO papers. Posting here and I'll try to keep it somewhat updated. Contributions welcomed!!! ❤️ 

## Pre 2020
* MARTA GANs: Unsupervised representation learning for remote sensing image classification (IEEE Geoscience and Remote Sensing Letters, 2017)
* Tile2vec: Unsupervised representation learning for spatially distributed data (AAAI, 2019)
* Learning to interpret satellite images using wikipedia (IJCAI, 2019)
* In-Domain Representation Learning for Remote Sensing (arxiv, 2019)

## 2021
* Seasonal Contrast: Unsupervised Pre-Training from Uncurated Remote Sensing Data (ICCV, 2021)
* Efficient visual pretraining with contrastive detection (ICCV, 2021)
* Self-Supervised Learning of Remote Sensing Scene Representations Using Contrastive Multiview Coding (CVPR, 2021)
* Representation Learning for Remote Sensing: An Unsupervised Sensor Fusion Approach (arxiv, 2021)
* Geography-Aware Self-supervised Learning (ICCV, 2021)
* Semantic segmentation of remote sensing images with self-supervised multitask representation learning (JSTARS, 2021)
* A generalizable and accessible approach to machine learning with global satellite imagery (Nature communications, 2021)
2022
* Global and local  contrastive  self-supervised learning for semantic segmentation of HR remote sensing images (IEEE * Transactions on Geoscience and Remote Sensing, 2022)
* SatMAE: Pre-training transformers for temporal and multi-spectral satellite imagery (NeurIPS, 2022)
* An Empirical Study of Remote Sensing Pretraining (Transactions on Geoscience and Remote Sensing, 2022)
* Timl: Task-informed meta-learning for agriculture (arxiv, 2022)

## 2023

* CMID: A Unified Self-Supervised Learning Framework for Remote Sensing Image Understanding (IEEE Transactions on Geoscience and Remote Sensing, 2023)
* SSL4EO-S12: A Large-Scale Multi-Modal, Multi-Temporal Dataset for Self-Supervised Learning in Earth Observation (IEEE Geoscience and Remote Sensing Magazine, 2023)
* CSP: Self-Supervised Contrastive Spatial Pre-Training for Geospatial-Visual Representations (ICML, 2023)
* Scale-MAE: A scale-aware masked autoencoder for multiscale geospatial representation learning (ICCV, 2023)
* SatlasPretrain: A Large-Scale Dataset for Remote Sensing Image Understanding (ICCV, 2023)
* Cross-Scale MAE: A Tale of multi-scale Exploitation in Remote Sensing (NeurIPS, 2023)
* GeoCLIP: Clip-Inspired Alignment between Locations and Images for Effective Worldwide Geo-localization (NeurIPS, 2023)
* SSL4EO-L: Datasets and Foundation Models for Landsat Imagery (NeurIPS Datasets and Benchmarks, 2023)
* A Billion-scale Foundation Model for Remote Sensing Images (arxiv, 2023)
L* ightweight, Pre-trained Transformers for Remote Sensing Timeseries (arxiv, 2023)
* Foundation Models for Generalist Geospatial Artificial Intelligence (arxiv, 2023)
* USat: A Unified Self-Supervised Encoder for Multi-Sensor Satellite Imagery (arxiv, 2023)
* DiffusionSat: A Generative Foundation Model for Satellite Imagery (arxiv, 2023)
* SpectralGPT: Spectral Foundation Model (arxiv, 2023)
* SatCLIP: Global, General-Purpose Location Embeddings with Satellite Imagery (arxiv 2023)
